webpack.tests.babel.js

```diff
@@ -50,13 +50,7 @@ module.exports = {
     ],
   },
   resolve: {
-    root: [
-      paths.docsRoot
-    ],
-    modulesDirectories: [
-      'node_modules',
-      '.'
-    ],
+    root: paths.root,
```

Comment:
Also allow root require in webpack dev server tests, though, this config will likely go away in favor of phantomjs only tests soon.